apis/python/HISTORY.md

```diff
@@ -20,6 +20,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - \[[#4177](https://github.com/single-cell-data/TileDB-SOMA/pull/4177)\] Update [TileDB core to 2.28.1](https://github.com/TileDB-Inc/TileDB/releases/tag/2.28.1).
 - Update TileDB version to https://github.com/single-cell-data/TileDB-SOMA/pull/4177
 - \[[#4209](https://github.com/single-cell-data/TileDB-SOMA/pull/4209)\], \[[#4220](https://github.com/single-cell-data/TileDB-SOMA/pull/4220)\] DataFrame columns of dictionary type, with a `large_string` or `large_binary` value type, were incorrectly reported as an Arrow `string`. They are now correctly reported as dictionary-typed fields with a value type of `large_string` and `large_binary`, respectively. NB: all string/binary types are automatically up-cast to their large variant in tiledbsoma.
+- \[[#4250](https://github.com/single-cell-data/TileDB-SOMA/pull/4250) Make X an optional argument for `ExperimentAxisQuery.to_anndata` in parity with `tiledbsoma.io.to_anndata`. \]
```

Comment:
```suggestion
- \[[#4250](https://github.com/single-cell-data/TileDB-SOMA/pull/4250) \] Make X an optional argument for `ExperimentAxisQuery.to_anndata` in parity with `tiledbsoma.io.to_anndata`.
```
The brackets should just be around the PR number. (You can see how it renders if you "view file").